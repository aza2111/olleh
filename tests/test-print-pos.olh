/*
 * Positive test: concat of string + int
 */

//String adj = "Printing a string with an int concatenated: ";
print  "Printing a string with an int concatenated: " + 5;
