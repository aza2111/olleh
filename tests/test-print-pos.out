Printing a string with an int concatenated: 5
